mountpoint-s3/src/fuse/session.rs

```diff
@@ -182,11 +193,17 @@ impl<W: Work> WorkerPool<W> {
         else {
             return Ok(false);
         };
-        self.state.idle_worker_count.fetch_add(1, Ordering::SeqCst);
+
+        let new_count = old_count + 1;
+        let idle_worker_count = self.state.idle_worker_count.fetch_add(1, Ordering::SeqCst) + 1;
+        metrics::gauge!(METRIC_NAME_FUSE_WORKERS_TOTAL).set(new_count as f64);
```

Comment:
Shouldn't this also be set in `start`?

Reply:
It is set when `start` is called, since `start` will invoke this method.